### ISSUE TITLE
Give PySyft and PyGrid version for every tutorial

### ISSUE THREAD

**Issue by Rene36** [OpenMined/PySyft#6205]
For me it is hard to follow the pace of changes in PySyft and PyGrid. There are plenty tutorials and blog posts available, however the majority does not work with v0.5.0 and it seems some already use v0.6.0.

Therefore, I propose to make it mandatory for every tutorial, jupyter notebook or blog post to give at least the version of PySyft, PyGrid/Hagrid and PyTorch. Also, every data set used should be directly connected to the tutorial so it is easy to find.

For example, with v0.5.0 the `sy.login(email="info@openmined.org", password="changethis", port=8081)` returns `TypeError: login() got an unexpected keyword argument 'port'`. So I assume the tutorial [here](https://github.com/OpenMined/PySyft/blob/dev/notebooks/course3/L2_Demo.ipynb) uses v0.6.0, but it is not given anywhere. Also, when starting hagrid and then connecting to it via the web browser briefly shows that v0.6.0alpha is used, but this information is either well hidden or not given at all. 

So please make it easier for others to use your framework without the hustle to dig into the source code to understand what I am doing wrong.

**Comment by rasswanth-s** [comment]
True , the blogposts and tutorials does not point to the exact version for which it was created, some are pretty old.It might be hard to work with them. 

In the future blog posts we would  pin the exact version for which it was created.

Currently at this point, the best starting point to learn more about syft ecosystem , would  be start at our new course.
**Remote Data Science** : https://courses.openmined.org/courses/introduction-to-remote-data-science.
which covers most of the new API,
Feel to post any questions.

Closing the issue for now, reopen if necessary.
